components/Head.js

```diff
@@ -9,7 +9,8 @@ export default function HeadComponent({
   description = 'JSON Schema Blog',
   image = '/img/social/card.png',  // Need to fix this!
   rssTitle = 'RSS Feed for the JSON Schema Blog',
-  rssLink = '/rss.xml'
+  rssLink = '/rss.xml',
+  canonicalLink = false
```

Comment:
ESLint is complaining about this line, but I'm not sure how to fix the error:

`'canonicalLink' is missing in props validation` eslint ([react/prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/prop-types.md))

Reply:
I wouldn't worry abou that. it's not the best but it does the job.